### ISSUE TITLE
Retrain your cnn?

### ISSUE THREAD

**Issue by tungdop2** [Rahul0128/Face-Mask-Detection-Pytorch#2]
Your pretrained resnet9 work pretty well, but in some case where the face is just 30 pixels or maybe 400 pixels, it does not. Should I retrain it for the small case and big case in particular?
